protocol/simplified-guardian.md

```diff
@@ -0,0 +1,118 @@
+# Simplified Guardian System
+
+## Purpose
+
+Provides a design for a simplified Guardian role that aligns the OP Stack with the latest update to
+[L2Beat's Stage 1 definition][1].
+
+## Context
+
+L2Beat recently published [an update][1] to the Stages Framework that is scheduled to take effect
+in the middle of 2025. The OP Stack is impacted by this update. Modifications will need to be made
+to guarantee that the OP Stack continues to be compatible with the Stage 1 definition.
+
+## Problem Statement
+
+L2Beat's updated definition specifically impacts ability to use the Deputy Guardian alongside the
+default Guardian role. The Deputy Guardian conflicts with the updated Stage 1 definition because it
+allows a role *other* than the Guardian (assumed to be held by a Security Council) to execute
+unilateral actions that cannot always be easily reversed by the Guardian.
+
+Specifically, L2Beat highlights the following problematic case for OP Mainnet:
+
+1. Deputy Guardian (held by Optimism Foundation) triggers the fallback to the
+  `PermissionedDisputeGame`.
+2. Deputy Guardian convinces a blocking minority of the Security Council by some means to *not*
+  switch back to the `PermissionlessDisputeGame`, to *not* challenge any of the dispute game
+  proposals created by the permissioned Proposer role, and to not trigger the Superchain-wide pause
+  mechanism.
+3. Permissioned Proposer role publishes an invalid proposal and the Security Council fails to
+  challenge the dispute game through any of its available mechanisms.
+4. Malicious party is able to execute an invalid withdrawal based on the invalid proposal.
+
+L2Beat's updated Stage 1 definition, once it goes into effect in the middle of 2025, would define a
+system that *allows* this situation as a Stage 0 system. We must therefore make adjustments to the
+incident response process so that OP Mainnet and other similarly configured systems remain Stage 1
+when this definition goes into effect.
+
+## Proposed Solution
+
+### Remove the Deputy Guardian
+
+We begin by removing the Deputy Guardian role entirely. The Guardian role would be held exclusively
+by the 1/1 Security Council Operations Safe (held by the Security Council). All Guardian actions
+would therefore only be executable by a majority of the Security Council. This eliminates the
```

Comment:
This statement is ambiguous about its implications:

> All Guardian actions would therefore only be executable by a majority of the Security Council. 

ie. given that the `pause()` will still be executable by the Foundation this implies that the pause is no longer a guardian action. I don't think that's the intent so: 

 ```suggestion
by the 1/1 Security Council Operations Safe (held by the Security Council). All Guardian actions other than `pause()`
would therefore only be executable by a majority of the Security Council. This eliminates the
```